src/paho/mqtt/client.py

```diff
@@ -2407,20 +2680,20 @@ def _packet_read(self):
             try:
                 command = self._sock_recv(1)
             except BlockingIOError:
-                return MQTT_ERR_AGAIN
+                return MQTTErrorCode.MQTT_ERR_AGAIN
             except ConnectionError as err:
                 self._easy_log(
                     MQTT_LOG_ERR, 'failed to receive on socket: %s', err)
-                return MQTT_ERR_CONN_LOST
+                return MQTTErrorCode.MQTT_ERR_CONN_LOST
             except TimeoutError as err:
                 self._easy_log(
                     MQTT_LOG_ERR, 'timeout on socket: %s', err)
-                return MQTT_ERR_CONN_LOST
+                return MQTTErrorCode.MQTT_ERR_CONN_LOST
             else:
                 if len(command) == 0:
-                    return MQTT_ERR_CONN_LOST
-                command, = struct.unpack("!B", command)
-                self._in_packet['command'] = command
+                    return MQTTErrorCode.MQTT_ERR_CONN_LOST
+                command_value, = struct.unpack("!B", command)
+                self._in_packet['command'] = command_value
```

Comment:
Or just
```suggestion
                self._in_packet['command'] = command[0]
```
since `command` is a `bytes`, and indexing into `bytes` gives you integers (and `B` doesn't need endianness):

```
>>> buf = b"@"
>>> struct.unpack("!B", buf)[0]
64
>>> buf[0]
64
```